input/pagecontent/reference.md

```diff
@@ -757,16 +838,16 @@ For other code systems, the value and units can also be set independently. To as
 * <Quantity>.value = {decimal}
 ```
 
-The units of measure can either be assigned a coded value or bound to a value set:
+The units of measure can be set by assigning a coded value to a Quantity:
 
-<pre><code>* &lt;Quantity&gt; = {CodeSystem name|id|url}#{code} <i>"{units display string}"</i>
+<pre><code>* &lt;Quantity&gt; = <i>{CodeSystem name|id|url}|{version string}</i>#{code} <i>"{units display string}"</i></code></pre>
 
-* &lt;Quantity&gt; from {ValueSet name|id|url}
-</code></pre>
+A Quantity can also be bound to a value set:
 
-In the first expression, `CodeSystem` corresponds to Quantity.system, `code` to Quantity.code, and `display string` to Quantity.unit.
+<pre><code>* &lt;Quantity&gt; from {ValueSet name|id|url}
+</code></pre>
 
-> **Note:** The ability to assign a coded value or bind a value set directly to a Quantity is a consequence of FHIR's definition of Quantity as a coded data type, rather than using a Coding to represent the units of measure.
+> **Note:** The ability to assign a coded value or bind a value set directly to a Quantity is a consequence of FHIR's definition of Quantity as a coded data type (probably not the greatest idea in the world).
```

Comment:
Woah this line isn't holding back on FHIR.

Reply:
Ok, you're right.  "The ability to assign a coded value or bind a value set directly to a Quantity is a consequence of FHIR oddly treating Quantity as a coded data type."